packer/marketplace-image.json

```diff
@@ -30,9 +31,34 @@
       "type": "shell",
       "inline": [
         "cloud-init status --wait",
+        "wget -qO - https://www.mongodb.org/static/pgp/server-4.0.asc | apt-key add -",
+        "wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -",
+        "apt -yy install apt-transport-https",
+        "apt-add-repository -y ppa:brightbox/ruby-ng",
         "apt -qqy update",
         "apt -qqy -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' full-upgrade",
-        "apt -qqy -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' install {{user `apt_packages`}}"
+        "apt -qqy -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' install {{user `apt_packages`}}",
+        "systemctl daemon-reload",
+        "systemctl enable elasticsearch",
+        "systemctl enable mongod",
+        "systemctl enable rails",
+        "systemctl enable sidekiq",
+        "systemctl start elasticsearch",
+        "systemctl start mongod",
+        "ruby-switch --set ruby2.6",
+        "gem install bundler --no-doc",
+        "mkdir -p /srv/shop",
+        "cd /srv/shop",
+        "bundle init",
+        "echo \"gem 'workarea', github: 'workarea-commerce/workarea', branch: 'v3.4-stable'\" >> Gemfile",
```

Comment:
This is necessary so that a working version of Sprockets is installed, once 3.4.19 gets released on Wednesday I can change this to `bundle add workarea`, hence why I opened this as a "draft PR".